internal/tofu/node_resource_plan_instance.go

```diff
@@ -554,6 +554,17 @@ func (n *NodePlannableResourceInstance) importState(ctx EvalContext, addr addrs.
 		return instanceRefreshState, diags
 	}
 
+	// Insert marks from configuration
```

Comment:
nit

This and the above parts (refreshing and failure in case of importing a non-existent object) are shared with the implementation of the import command as well.
Consider extracting them to some common function `refreshImportedState`, which would refresh + error on non-existent remote object + add schema marks.

Alternatively, just extract the "add schema marks" part to a common function

Reply:
Good suggestion, and I prefer the latter. 
I am not sure how will the plannable part differ to the import command, with we add more feature to plannable import(such as support for_each). 
So extract the "add schema marks" part to a common function is good enough for now.